src/ast/return-statement.ts

```diff
@@ -0,0 +1,20 @@
+import Context from '../semantics/context';
+import { AstNode } from '../type-definitions/plainscript-types';
+
+export default class ReturnStatement extends AstNode<ReturnStatement> {
+  constructor(public returnValue: any) { super(); }
```

Comment:
`returnValue` should be typed `Expression` or `Expression|null` I think

Reply:
thanks !